packages/wallet-adapter-react/src/WalletProvider.tsx

```diff
@@ -75,28 +76,33 @@ export const AptosWalletAdapterProvider: FC<AptosWalletProviderProps> = ({
   };
 
   const signAndSubmitTransaction = async (
-    transaction: Types.TransactionPayload
+    transaction: Types.TransactionPayload,
+    options?: TransactionOptions | undefined
```

Comment:
no need the `undefined` as this is an optional param

Reply:
Thanks, commit [6fd7c33](https://github.com/aptos-labs/aptos-wallet-adapter/pull/156/commits/6fd7c3390ead72f912a95e1080538f5bb79e2d8f)